library/src/main/java/com/bumptech/glide/load/engine/exceptions/ErrorWrappingGlideException.java

```diff
@@ -0,0 +1,11 @@
+package com.bumptech.glide.load.engine.exceptions;
```

Comment:
Yes, but we'll need to forward-port this change it to v4, otherwise v3 and v4 diverges.
And if we keep this exception when forward-porting it would make sense to have them in the same package from the beginning.